po/glossary/pt.po

```diff
@@ -210,14 +208,12 @@ msgid "balance (noun)"
 msgstr "saldo"
 
 #. "Balance brought forward - usually the first entry of an account statement containing the 'balance c/f' of the previous billing period or page"
-#, fuzzy
 msgid "balance b/f"
-msgstr "reconciliar"
+msgstr "saldo"
 
 #. "Balance carried forward - usually the last entry of an account statement to be used as 'balance b/f' on the next billing period or page"
-#, fuzzy
 msgid "balance c/f"
-msgstr "reconciliar"
+msgstr "saldo"
```

Comment:
Isn't there a difference between the saldo at the beginning and the end of the account statement?

Reply:
According to the comments, I believe it is the value at the end of, e.g. a month that passes to the next month. Beginning balance and ending balance. So no, the word should be the same. I could add "inicial" and "final"?

Reply:
> I could add "inicial" and "final"?

Sounds good.